### PR TITLE
VZ-8175: Implement container security context for jaeger components when Jaeger is upgraded to 1.38.0 or above

### PR DESCRIPTION
**Pull request by prasad-shirodkar**
Implement container security context for jaeger-operator and jaeger components when Jaeger is upgraded to 1.38.0 or above

### CODE REVIEW

File: platform-operator/thirdparty/manifests/jaeger/jaeger-operator.yaml
Comment (ayaranga) [inline platform-operator/thirdparty/manifests/jaeger/jaeger-operator.yaml:1]: Why is this file required to be added?

Comment (prasad-shirodkar) [reply platform-operator/thirdparty/manifests/jaeger/jaeger-operator.yaml:1]: I updated this file accidentally when troubleshooting local installation. I removed the jaeger operator yaml file since it's not really referenced any longer - it is a remnant of when verrazzano was not using jaeger operator.

**Review by ayaranga — APPROVED** [review]
